types/dsinjs__binary-tree/tslint.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Imposing additional linting constraints on an individual package tends to make it harder for others to contribute.  Are these important?

Reply:
Which of these addition constraints do you recommend removing?

Reply:
Generally speaking, the contents of this file should be exactly `{ "extends": "dtslint/dt.json" }`.

Reply:
I see, thank you for your kind reply